blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/NodeDownstreamBuildAction.java

```diff
@@ -0,0 +1,54 @@
+package io.jenkins.blueocean.service.embedded.rest;
+
+import hudson.model.InvisibleAction;
+import hudson.model.Run;
+import org.jenkinsci.plugins.workflow.actions.FlowNodeAction;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+
+import java.util.Objects;
+
+/**
+ * Annotates a FlowNode to point to a downstream build triggered by said node. Applied by
+ * io.jenkins.blueocean.listeners.DownstreamJobListener in blueocean-pipeline-api-impl
+ */
+public class NodeDownstreamBuildAction extends InvisibleAction implements FlowNodeAction, BluePipelineAction {
```

Comment:
It could also be `BlueActionProxy`. Consistent with how we are handling actions and presenting them as resources.